site/en/docs/devtools/settings/other/index.md

```diff
@@ -0,0 +1,262 @@
+---
```

Comment:
I would like to know your thoughts on why making a docs for all other tabs vs using one docs per tab.

I have slight preference on using one docs per tab as it's clearer to search and link it to devtools in the future.

Reply:
I'm mostly concerned with the number of nested layers vs leaf (end article) granularity of our Information Architecture. And overall maintanability. Your point is valid, though.

I'll retire /docs/devtools/customize/dark-theme/ and /docs/devtools/customize/placement/ as you suggested and see if I can move this up a level in the hierarchy. I think the "Settings and customization" umrella section is redundant. It can be two sections at the same top level: "Settings" and "customization", where one is a reference and the other a collection of cosmetic options.

Reply:
I'll do it in this PR. It won't block the review, just a small restructuring.